changes/5880.fixed

```diff
@@ -0,0 +1,4 @@
+Fixed overflowing device component tables in device and module component tabs.
```

Comment:
Any chance a similar fix is needed for VirtualMachine/VMInterface?

Reply:
That one is already good. Thanks for the reminder to check it!